config/longrun_configs/longrun_aquaplanet_dyamond.yml

```diff
@@ -8,7 +8,7 @@ implicit_diffusion: true
 approximate_linear_solve_iters: 2
 topography: "Earth"
 start_date: "20160801"
-initial_condition: "artifact\"DYAMOND_SUMMER_ICS_p98deg\"/DYAMOND_SUMMER_ICS_p98deg.nc"
```

Comment:
```suggestion
```
Since the full resolution initial condition artifact is large, we can leave the `0.98 deg` resolution artifact as the default. The user change this as needed.